### ISSUE TITLE
[bugf][link-names][add discernible names to social links]

### ISSUE THREAD

**Issue by kyegomez** [The-Swarm-Corporation/swarms-website#25]
Lighthouse accessibility: Links do not have a discernible name. Failing elements: Discord, Twitter, and X Community social links use empty anchor tags. Add aria-label or visible text for screen reader users.
